scrapy/code/groceries/groceries/seleniumHelpers.py

```diff
@@ -0,0 +1,20 @@
+from scrapy_selenium import SeleniumRequest
+
+def create_unfiltered_parse_request(url,callback,wait_until):
```

Comment:
It looks like this method might be the same for multiple stores, i.e. Safeway, Wegmans, etc. And the method seems pretty simple. Is this something you might want to inherit from a parent class?

Reply:
Right now I was using them as static methods. Currently its just 2 functions in a file, maybe someday it will end up in a library